addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -244,7 +244,7 @@
                     <span class="d-none d-sm-inline-block">Next</span> <i class="fa fa-chevron-right ml-2"></i>
                 </a>
             </div>
-            <a t-if="is_training_channel" class="btn btn-light border ml-2" role="button" t-att-href="'/slides/slide/%s?fullscreen=1' % (slug(slide))">
```

Comment:
then you can remove the `<t t-set="is_training_channel"` declare above in the template